### PR TITLE
Add element-separator

### PR DESCRIPTION
**Pull request by mrdrogdrog**
This PR adds a component, which can place an react element between its children.

### CODE REVIEW

**Comment by mrdrogdrog** [comment]
An example usage could be
```
<ElementSeparator separator={<hr className="w-100 bg-white"/>}>
    {emailForm}
    {ldapForm}
</ElementSeparator>
```

File: src/components/element-separator/element-separator.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, {Fragment} from "react";
+
+export interface HrSeparatorProps {
```
Comment (DerMolly) [inline src/components/element-separator/element-separator.tsx]:
```suggestion
export interface ElementSeparatorProps {
```

File: src/components/element-separator/element-separator.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    separator: React.ReactElement
+}
+
+export const ElementSeparator: React.FC<HrSeparatorProps> = ({children, separator}) => {
```
Comment (DerMolly) [inline src/components/element-separator/element-separator.tsx]:
```suggestion
export const ElementSeparator: React.FC<ElementSeparatorProps> = ({children, separator}) => {
```

File: src/components/element-separator/element-separator.tsx (hunk 1 above)
Comment (mrdrogdrog) [reply src/components/element-separator/element-separator.tsx]: Already done.

File: src/components/element-separator/element-separator.tsx (hunk 2 above)
Comment (mrdrogdrog) [reply src/components/element-separator/element-separator.tsx]: Already done.

**Pull request merged**